src/codemodder/codemods/api/__init__.py

```diff
@@ -122,3 +122,16 @@ def leave_Call(self, original_node: cst.Call, updated_node: cst.Call):
                 return new_node
 
         return updated_node
+
+    def leave_Assign(self, original_node, updated_node):
```

Comment:
Adding individual handlers like this does not feel like a viable long-term solution.

Reply:
The goal is that we're going to get to a point that we have all of these cases. Right now most (all?) our api based codemods related to a func/method call so  leave_Call is appropriate. Your case is new so we need this. I think the django debug / session cookies codemods are similar and maybe we need to move those to the new api too

Reply:
I'm hoping that we can handle this via clever metaprogramming somehow (maybe with `__getattr__`?) rather than having to override each possible method explicitly.

Reply:
Yes, can be done with metaprogramming. Either use the `__dict__` directly (and recuresivelly over all the `mro()` classes, or use the `getmembers` from the `inspect` module.

Reply:
Yes that is another possibility. There are a variety of ways to approach this problem.